### PR TITLE
add 2019.3.11 update

### PR DESCRIPTION
**Pull request by flowerornament**
@rmariani can you give this a quick once-over?

### CODE REVIEW

**Comment by rmariani** [comment]
Yup

File: static-site/posts/updates/2019.3.11-update.udon [hunk 1]
```diff
@@ -0,0 +1,31 @@
+:-  :~  date/'~2019.3.20'
+        title/'~2019.3.11 Update'
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.11-update.udon:2]: I don't think this date is right. Shouldn't it be `~2019.3.20` In all instances?

File: static-site/posts/updates/2019.3.11-update.udon [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## `@p` Bug
+
+The `@p` bug [is nearly solved](https://github.com/urbit/arvo/pull/1110), thanks to [@max19](https://github.com/urbit/arvo/issues/1105#issuecomment-472585937) and *Jared* `~nidsut-tomdun`.
+
+## Team 
+
+*Ted* `~rovnys-ricfer`, *Philip* `~wicdev-wisryt`, and *Joe* `~master-morzod` developed a proposal for formalizing the semantics around state persistence in a way that enables parallelism, event replay, and a new model for error handling. *Joe* `~master-morzod` worked on integrating new `%eyre`. *Elliot* `~ponnys-podfer` worked on an elaborate proposal for resolving the `@p` bug involving constraint solvers, which turned out to be unnecessary, but was quite impressive nonetheless!
+
+*David* `~rapfyr-diglyt`, *Robert* `~lodleb-ritrul`, and 23 students completed the first week of Hoon School. `~hadrud-lodsef` implemented automation for city onboarding. In the next few weeks we're looking forward to automating Urbit support, the Hoon School queue, and contributor onboarding.
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.11-update.udon]: Did you forget to mention Will, or did he ask not to have his real name mentioned? Also, I don't think anyone will understand what "city onboarding" means, so maybe quickly explain it somehow.

File: static-site/posts/updates/2019.3.11-update.udon [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+;img@"https://media.urbit.org/site/posts/updates/~2019.3.11-update-1.jpg"(width "100%");
+
+## `@p` Bug
+
+The `@p` bug [is nearly solved](https://github.com/urbit/arvo/pull/1110), thanks to [@max19](https://github.com/urbit/arvo/issues/1105#issuecomment-472585937) and *Jared* `~nidsut-tomdun`.
+
+## Team 
+
+*Ted* `~rovnys-ricfer`, *Philip* `~wicdev-wisryt`, and *Joe* `~master-morzod` developed a proposal for formalizing the semantics around state persistence in a way that enables parallelism, event replay, and a new model for error handling. *Joe* `~master-morzod` worked on integrating new `%eyre`. *Elliot* `~ponnys-podfer` worked on an elaborate proposal for resolving the `@p` bug involving constraint solvers, which turned out to be unnecessary, but was quite impressive nonetheless!
```
Comment (rmariani) [inline static-site/posts/updates/2019.3.11-update.udon]: Elliot is still `~littel-ponnys`, I just asked.

File: static-site/posts/updates/2019.3.11-update.udon (hunk 1 above)
Comment (flowerornament) [reply static-site/posts/updates/2019.3.11-update.udon:2]: We date the weekly update to the week that it's documenting (week of ~2019.3.11). I could be convinced that this is confusing, but it's the pattern.

**Pull request merged**